cli/commands.go

```diff
@@ -71,7 +71,7 @@ func Commands(metaPtr *meta.Meta) map[string]cli.CommandFactory {
 				CredentialBackends: map[string]logical.Factory{
 					"approle":  credAppRole.Factory,
 					"cert":     credCert.Factory,
-					"aws-ec2":  credAwsEc2.Factory,
```

Comment:
I think to not break scripts that are setting up backends automatically we should actually support both `aws-ec2` and `aws` but only document the latter. @vishalnayak thoughts?